pages/api/program/index.ts

```diff
@@ -20,18 +20,19 @@ export default async function handle(
             break;
         }
         case "POST": {
-            // TODO:
-            if (!validateCreateProgram(req.body as createProgramInput)) {
-                ResponseUtil.returnBadRequest(
-                    res,
-                    `Required field(s) were not provided correctly.`,
-                );
+            const validationError = validateCreateProgram(
+                req.body as createProgramInput,
+            );
+            if (validationError.length !== 0) {
+                validationError[0] =
```

Comment:
what's the reason for this line?

Reply:
If the length of the list is 0, it means there were no errors when validating the request.
The next line was really small thing I added, the error phrases are all lower case and get joined, it makes the first character capitalized - not sure if it's necessary at all